ROADMAP.md

```diff
@@ -1,28 +1,27 @@
 # Roadmap
 
 This document defines a high level roadmap for Rook development and upcoming releases.
-The features and themes included in each milestone are optimistic in the sense that many do not have clear owners yet.
+The features and themes included in each milestone are optimistic in the sense that some do not have clear owners yet.
 Community and contributor involvement is vital for successfully implementing all desired items for each release.
 We hope that the items listed below will inspire further engagement from the community to keep Rook progressing and shipping exciting and valuable features.
 
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
-We use the [milestone](https://github.com/rook/rook/milestones) feature in Github so look there for the most up-to-date and issue plan.
+We use the [project boards](https://github.com/rook/rook/projects) in Github so look there for the most up-to-date issues and their status.
 
 
-## Rook 1.4
+## Rook 1.5
 
-The following high level features are targeted for Rook v1.4 (July 2020). For more detailed project tracking see the [v1.4 board](https://github.com/rook/rook/projects/18).
+The following high level features are targeted for Rook v1.5 (November 2020). For more detailed project tracking see the [v1.5 board](https://github.com/rook/rook/projects/19).
 
 * Ceph
```

Comment:
can we also add a list for NFS? @prksu has been doing a lot of work this summer for GSoC that will be included in 1.5, e.g. this list of PRs: https://github.com/rook/rook/pulls?q=is%3Apr+author%3Aprksu+

Highlights since 1.4 was tagged early August:
* A quota for disk usage can now be set and enforced for each NFS PV (https://github.com/rook/rook/issues/5788)
* NFS PV's can now be deleted and cleaned up (https://github.com/rook/rook/issues/3074)
* Isolation is now enforced between each PV provisioned for a given NFS export: (https://github.com/rook/rook/issues/4982)